<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStuent</class>
 <widget class="QDialog" name="AddStuent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="name_2">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="department_2">
         <property name="text">
          <string>Department</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="sex">
         <property name="text">
          <string>sex</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="DateOfBirth_1">
         <property name="text">
          <string>Date of Birth</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="name"/>
       </item>
       <item>
        <widget class="QLineEdit" name="department"/>
       </item>
       <item>
        <widget class="QRadioButton" name="SexMale">
         <property name="text">
          <string>Male</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="SexFemal">
         <property name="text">
          <string>Femal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="DateOfBirth"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="Cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Add">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
